build.sbt

```diff
@@ -2,14 +2,8 @@ val catsVersion = "2.7.0"
 val finagleVersion = "21.8.0"
 
 ThisBuild / tlBaseVersion := BaseVersion(finagleVersion)
-ThisBuild / tlMimaPreviousVersions := Set.empty
-
-// Finagle releases monthly using a {year}.{month}.{patch} version scheme.
-// The combination of year and month is effectively a major version, because
-// each monthly release often contains binary-incompatible changes.
-// This means we should release at least monthly as well, when Finagle does,
-// but in between those monthly releases, maintain binary compatibility.
-ThisBuild / versionScheme := Option("year-month-patch")
+ThisBuild / tlVersionIntroduced := // test bincompat starting from the beginning of this series
+  List("2.12", "2.13").map(_ -> s"${tlBaseVersion.value}.0").toMap
```

Comment:
This is a mild abuse of the `tlVersionIntroduced` setting, which is used to indicate in which version artifacts first existed.

Since we only want to check bincompat with in a `year.month.x` series (and not `year.x.y` series) all we have to do is pretend the artifacts were only introduced in version `year.month.0` :)

Reply:
Oh wait, this won't work, because sbt-typelevel won't look further back than the major version bump. Totally my bad, sorry.